### DISCUSSION TITLE
Questions for Promotor

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#37] [Q&A] [answered]
Questions for Promotor:

**Research Question:**
What are, for an ISV, the success factors of EA for AF in the public sector market?

The research question can go into two directions. How must EA function in a AF world? Or how can EA be used (artefacts) to become AF?

**Systems question:**

A lot of available theories in the literature are around IT Systems and not about EA or an organisation as a System. But if you really take a look at the most theories they should be universal on all systems (lens of systems theory). What if I proof (through delphi) that some of those theories are also applicable on other types of systems to become AF? Is this a viable approach? Think about diversity (could be smaller diverse organisational units that deliver roughly the same functions), loosly coupled (could be corporate startups, different types of information models loosly, etc..) 

**Artefact Question:**
With how must EA function in an AF world could have an outcome of a capability model (if you take a look at the three schools of EA by LaPalme it looks like that the third school (learning and external context) is more suitable for AF but what are those Capabilities needed?
With how EA can be used (succesfully) to be more AF could be a framework of validated principles. But this is more prescriptive (and steers on the stability of seneca Barbbell) but is kind of contra on that of the 3rd school of EA of LaPalme.

**Status for the master project coaching masterclass end of september**
My workstyle firstly synthesise as much as possible and after that storm writing. Research Methodology, Background and Intro will be in concept on the master project coaching day on the AMS. How should I present this? Just open the PDF and walk things through, also take a look at the planning and show our fellow students the progress?

**Process of Promotor & Co-Promotor**
Who does what and when?
Proposal: 

- Edzo for day to day operations and theory
- Hans for directions on crossroads
- Edzo has access to GitHub does Hans also want to have this?
- Proposal is every time a good piece of work is being done a review copy to Hans? Edzo will follow more closely?
- Research Log is on GitHub (wiki)
- Discussions and Questions are noted on the discussion part of GitHub (everything is traceable)

**Tooling:**
How to get access to meetingwizard? Going to use this for the questionaires of the delphi groups.

**Status on Delphi**
- Academics are selected. Only one did not accept (still have three left and that is enough).
- ISV is almost finished need two more for another ISV (will be known next week)
- We dropped the idea of VNG (Maarten and I) They do not have the people on staff. So we do see VNG the same as municipalities. Working on an introduction email for Maarten to start inviting from his network (Logius, BZK, Municipalities).
- Already invited the EA of de Drechtsteden to participate. He accepted (first municipality).
- Also thinking of asking Tax Offices their world is also changing all the time (is also local gov).

**Comment by JRBliekendaal** [comment, marked as answer]
Answers:
**On Research Question**
Take the original research concept model... EA delivering AF.. what are those success factors.. and not how should EA be (capabilities for example).

**Systems Question**
Is possible but with proof like with the use of other theories (if x is y then y is x). DEMO can be used (example). All depends also on the lens that is choosen.
@edzob I have the feeling that I miss something here. Hate it that I did not take the notes directly. Do you still have the choosen direction in mind?

**Artefact Question**
Do not forget that there are two sides of the barbbell. Maybe prescriptive (intentional) for the core stability and the other side of the barbbell other things (those are the successfactors).

**Status for the master project coaching masterclass end of September**
Mentioned. And agreed.

**Tooling**
Agreed with Promotor on the use of Meetingwizard. AMS has two concurrent licenses. When Meetingwizard is required Just send an email. Process is that you first will get a tour on the tool before you can use it.

**Process of Promotor & Co-Promotor**
Approach agreed upon.
Addon from Promotor. Once a month a status meeting with Promotor, Co-Promotor and Masters Candidate.

Tip/Hint from Promotor. Request sponsor to take a position on the jury in the defense.

**Comment by edzob** [comment]
>>Systems Question
>>Is possible but with proof like with the use of other theories (if x is y then y is x). 
>>DEMO can be used (example). All depends also on the lens that is choosen.
>>@edzob I have the feeling that I miss something here. 
>> Hate it that I did not take the notes directly. Do you still have the choosen direction in mind?

Yes..
Hans Mulder told:
if you use the lens of DEMO then you can not state that if a principle works for the data layer then it also works for social layer. 
So adopting the ontology of DEMO would prefent this.

If you use a different lens as for example that of Normalized Systems Theory, then there principles that apply to the IT domain and also apply to the social domain. Jan Verelst is collecting evidense that the NST principles not only apply to code and programm interfaces but also to physical objects, business processes and law.

So if you want to re-use principles from the IT architecture domain into the organisational domain, you need to support this by applying the right lens and validation of the application of this lens

**Reply by JRBliekendaal** [reply]
@edzob  thx... also talked to hans today and he told me the same again ;) thx!!! so not w third and the triangulation is executed 😁🤣😎

**Reply by edzob** [reply]
double check for all the things!! :)
